Tribute.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name = 'Tribute'
+  s.version = '1.0'
```

Comment:
I wouldn't be so optimistic – I don't know if we can guarantee API backward compatibility just yet🙂